codes/javascript/chapter_computational_complexity/time_complexity.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * File: time_complexity.js
+ * Created Time: 2023-01-02
+ * Author: RiverTwilight (contact@rene.wang)
+ */
+
+class time_complexity {
+	/* 常数阶 */
+	constant(n) {
+		var count = 0;
```

Comment:
`let` is preferrable.

Reply:
In many cases, the count is dynamic (for example, excuting`count++`) so I choose `var`. As for let or var, I only use `var` when the variable is at the top level of a function but actually a personal habit I can of course change them all here:D
![image](https://user-images.githubusercontent.com/52880665/210230891-1aedaf22-04bf-4f70-a69a-5dd5aeb48bc1.png)

Reply:
More information at [understanding-let-const-and-var-keywords](https://www.freecodecamp.org/news/understanding-let-const-and-var-keywords/).

In short:

That's all, my friends. You need to consider these circumstances and concepts to evaluate how var, let, and const behave. So, the rule goes:

Don't use var anymore.
Use let or const.
Use const more often. Use let when you need to reassign another value to a variable.
Don't try to access a variable without declaring it.